boltdesignsystem/bolt
# DS-265 Bolt-card action buttons fix

## Jira

https://pegadigitalit.atlassian.net/browse/DS-265

## Summary

Action buttons start to rendered wrong, see image below

![image](https://user-images.githubusercontent.com/16049049/100092917-1c7e3e00-2e57-11eb-8ab5-849530c66d3a.png)

## Details

Update the code that was pushed with some icons updates.

## How to test

Go to the `/pattern-lab/patterns/40-components-card-replacement-999-card-replacement-with-web-component/40-components-card-replacement-999-card-replacement-with-web-component.html` and check if the buttons are rendered correctly.

![image](https://user-images.githubusercontent.com/16049049/100093086-4f283680-2e57-11eb-9500-acd0d4ab575e.png)

## Release notes

(Optionally, add details that will be added to the release notes. Use this to call out deprecations. [Read more](https://intuit.github.io/auto/docs/generated/changelog#additional-release-notes))
